heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -481,7 +521,7 @@ public FullQuery end(int resolved, int failed, int cancelled) throws Exception {
                     checkIssues(failed, cancelled).orElse("Query exceeded quota")));
 
                 return new FullQuery(trace, errorsBuilder.build(), ImmutableList.of(),
-                    Statistics.empty(), new ResultLimits(limitsBuilder.build()));
+                                     Statistics.empty(), new ResultLimits(limitsBuilder.build()));
```

Comment:
Could possibly using the wrong style config?
https://github.com/spotify/heroic/tree/master/idea